content/doc/tutorials/using-jenkins-to-build-a-java-maven-project.adoc

```diff
@@ -0,0 +1,321 @@
+---
+layout: documentation
+title: Using Jenkins to build a Java/Maven project
+section: doc
+---
+
+:toc:
+:imagesdir: /doc/book/resources
+
+This tutorial shows you how to automate building a simple Java project (built
+with Maven) using Jenkins.
+
+If you are a Java/Maven developer who is new to CI/CD concepts, or you might be
+familiar with these concepts but don't know how to implement building your
+project using Jenkins, then this tutorial is for you.
+
+*Duration:* This tutorial takes 20-40 minutes to complete (assuming you've
+already met the <<prerequisites,prerequisites>> below). The exact duration will
+depend on the speed of your machine and whether or not you've already
+<<download-and-run-jenkins-in-docker,downloaded and run Jenkins in Docker>> from
+link:..[another introductory tutorial].
+
+You can stop this tutorial at any point in time and continue from where you left
+off.
+
+include::doc/tutorials/_prerequisites.adoc[]
+
+include::doc/tutorials/_download-and-run-jenkins-in-docker.adoc[]
+
+
+=== Fork and clone the sample repository on GitHub
+
+In this tutorial, we'll be building a simple Java application using Maven, whose
+overall build process will be orchestrated in Jenkins. This Java application
+(which you'll obtain from a sample repository on GitHub) simply outputs the
+string "Hello world!" and is accompanied by a unit test to check that the main
+application works as expected.
+
+. Ensure you are signed in to your GitHub account. +
+  *Tip:* If you don't yet have a GitHub account, sign up for a free one on the
+  https://github.com/[GitHub website].
+. Fork the
+  https://github.com/gilesgas/simple-java-maven-app[`simple-java-maven-app`] on
```

Comment:
I think we should pick a naming convention like "sample-*" or "tutorial-*" and put this repository into `jenkins-infra`.

Reply:
Thanks @bitwiseman - I like the idea of adding "_tutorial-_" as a prefix, so I'll gather you're suggesting we name these repos something like "_tutorial-simple-java-maven-app_" or just "_tutorial-java-maven-app_"?

I guess there's also no way to group repos already within the "jenkins-infra" organization on GitHub (and hence the prefix suggestion)?